winter-framework/src/main/java/de/uni_mannheim/informatik/dws/winter/datafusion/AttributeValueFuser.java

```diff
@@ -168,28 +165,34 @@ public AttributeValueFuser(ConflictResolutionFunction<ValueType, RecordType, Sch
 	 */
 	protected FusedValue<ValueType, RecordType, SchemaElementType> getFusedValue(RecordGroup<RecordType, SchemaElementType> group, Processable<Correspondence<SchemaElementType, Matchable>> schemaCorrespondences, SchemaElementType schemaElement) {
 		List<FusibleValue<ValueType, RecordType, SchemaElementType>> fusableValues = getFusableValues(group, schemaCorrespondences, schemaElement);
+		
 		FusedValue<ValueType, RecordType, SchemaElementType> fusedValueInstance = conflictResolution.resolveConflict(fusableValues);
 		
 		// Collect fusion results for debugging purposes!
-		if(fusableValues.size() != 0 && fusedValueInstance.getValue() != null){
+		if(this.isCollectDebugResults() && fusedValueInstance.getValue() != null){
 			
-			TableRow resultsRow = new TableRow(this.getDebugFusionResults().getSize() + 1, this.getDebugFusionResults());
+			String identifiers =  schemaElement.getIdentifier() + "-";
+			String values = "";
 			
-			String[] identifiersArray = new String[fusableValues.size()];
-			String[] valuesArray = new String[fusableValues.size()];
 			for(int i = 0; i < fusableValues.size(); i++){
-				identifiersArray[i] = fusableValues.get(i).getRecord().getIdentifier().toString();
-				valuesArray[i] = fusableValues.get(i).getValue().toString();
+				if(i == 0){
+					identifiers += fusableValues.get(i).getRecord().getIdentifier().toString();

```

Comment:
please use de.uni_mannheim.informatik.dws.winter.webtables.ListHandler to format the identifier and value lists as string

Reply:
Done